src/main/resources/labels/televisions-electronic-displays/electronic-displays-2021.svg

```diff
@@ -2,11 +2,11 @@
   <defs>
     <style>
       #hdrSection {
-        visibility: hidden;
+        display: none;
```

Comment:
For some reason the PDF render didn't apply the `visibility: hidden` properly here. It would still show the child text content even though the group was hidden. 
![image](https://user-images.githubusercontent.com/3612419/95756779-2597bf00-0c9e-11eb-88a4-d8b62904b318.png)

Using `display: none` instead works, and it didn't look like any of the rest document got displaced by doing this. But would be good if you could double check.

Reply:
Ah yes I came across this too, it looks like the renderer doesn't apply `visibility: hidden` to descendants. `display: none` is fine